packages/components/src/components/dialog/dialog.stories.ts

```diff
@@ -1,20 +1,30 @@
 import '../../solid-components';
 import { html } from 'lit-html';
 import { storybookDefaults, storybookHelpers, storybookTemplate } from '../../../scripts/storybook/helper';
-import { waitUntil } from '@open-wc/testing-helpers';
 import { withActions } from '@storybook/addon-actions/decorator';
 
 const { overrideArgs } = storybookHelpers('sd-dialog');
 const { argTypes, parameters } = storybookDefaults('sd-dialog');
 const { generateTemplate } = storybookTemplate('sd-dialog');
 
 /**
- * **Disclaimer** : Input-based controls for `sd-dialog` are unavailable on this Docs page due to a [Storybook bug](https://github.com/solid-design-system/solid/issues/702). Please use the individual feature stories to access these controls.
+ *
+ * Used over other content. It requires an interaction from the user before they can return to whatever is underneath.
+ *
+ * - Use the `headline` slot to add a headline.
+ * - Use the `default` slot to add main content. <br /> Default slot is always scrollable.
+ * - Use the `footer` slot to add action elements. <br /> Footer slot is always fixed.
+ *
+ *
+ *  **Related templates**:
```

Comment:
For the 1st preview of Dialog can we make it more similar to what we have in FIGMA?
"Footer slot" is the same size as other slots in FIGMA